src/migrations/20230222154915-create-notiications-table.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            create table if not exists notifications (
```

Comment:
I'd keep the SQL elegant by splitting UPPECASE stuff from lowercase like here: https://github.com/Unleash/unleash/blob/main/src/migrations/20230208093942-assign-project-api-token-permissions-member.js